Atomic/sign.py

```diff
@@ -72,6 +72,9 @@ def no_reg_no_default_error(image, registry_path):
                     if signature_path is None:
                         raise ValueError("No write path for {}/{} was "
                                          "found in {}".format(reg, repo, registry_config_path))
+                    elif signature_path.startswith("http"):
+                        raise ValueError("Writing to {} is not supported. Use a "
+                                         "file:///location instead.".format(signature_path))
 
                     # Deal with write path prepends
                     if signature_path.startswith("file://"):
```

Comment:
If I understand  correctly, that still seems brittle and semantically incorrect; if the user _did_ for some crazy reason have a local directory tree named `http://example.com/sigstore`, we _still_ wouldn’t want to use that local path.

Use real URL parsing, and extract URL schema first. _If_ it is a `file:`, sure, then we can do an `os.path.exists() ` to be friendly. But we should be refusing to write to a `http` regardless of how reasonable or crazy the local filesystem contents are.